### PR TITLE
feat: per-install ComfyUI-Manager security level + network mode in Startup Args

### PR DESCRIPTION
**Pull request by Kosinkadink**
## Summary

Gives Desktop per-install UI controls for ComfyUI-Manager's `security_level` and `network_mode`, addressing part of #1037.

ComfyUI-Manager (v4) has no runtime API to change these - it reads both values once from `config.ini` at startup. As a result there is no way to relax or tighten which node packs may be installed, and the restriction is invisible to the user. The `--listen` case is the motivating one: with a non-loopback `--listen`, Manager denies node pack installs at EVERY `security_level` unless `network_mode = personal_cloud` (`To use this action, security_level must be 'normal or below', and network_mode must be set to 'personal_cloud'`). Since Manager won't own these controls, Desktop does.

## Changes

- Add per-install `managerSecurityLevel` (`strong` / `normal` / `normal-` / `weak`, default `normal`) and `managerNetworkMode` (`public` / `private` / `offline` / `personal_cloud`, default `public`) fields stored on the installation record.
- Expose both as selects in each local install's **Startup Args** tab (picker drawer), side-by-side on one row, with per-option descriptions and tooltips explaining the `--listen` implications (wording checked against Manager v4's actual `is_allowed_security_level` gate). Remote/cloud sources do not show them. Marked `requiresRestart`.
- New `rowGroup` field mechanism renders adjacent same-group fields as one responsive paired row; also applied to **Launch Mode + Browser Cache** to keep the tab compact and leave more room for env-var rows.
- Reconcile `config.ini` before each local launch using the launched install's own values:
  - fresh installs: write both options (plus the China-mirror keys when opted in),
  - existing configs: update the `[default]` options in place while preserving all other keys, mirroring Python `ConfigParser(strict=False)` semantics: case-insensitive option keys, both `=` and `:` delimiters, duplicate collapse across repeated `[default]` sections (later duplicates win in Manager, so all must collapse), section headers with trailing text, and removal of indented value-continuation lines,
  - legacy-only Manager configs are edited in place (avoids tripping Manager's legacy-migration path),
  - invalid stored values degrade to the default,
  - one read + at most one write even when both options are reconciled.
- **Fail closed:** when the user chose a Manager option and the config write fails, the launch is aborted with an actionable error instead of silently running Manager with stale (possibly weaker) security settings. Mirror-only seeding keeps the old non-blocking behavior.
- `useChineseMirrors` remains a global setting.
- Generalize `ensureManagerMirrorConfig` into `ensureManagerConfig`.
- Add en/zh translations.

## Companion PR

The offline-loading and security-message UX fixes from #1037 are handled in ComfyUI_frontend on branch `fix/manager-offline-loading-and-security-msg`.

## Testing

- `pnpm typecheck` / `pnpm lint` / `pnpm build` / `pnpm test` pass (194 files, 2913 tests).
- Unit coverage: `managerConfig` (INI reconciliation edge cases including colon delimiters, repeated `[default]` sections, continuation lines, mixed-case duplicates), `managerConfigLaunch` (per-install pass-through, invalid degrade, remote skip, fail-closed on write failure vs. non-blocking mirror seeding), `launchSettingsFields` (select shape, labels, option order, per-record independence, row pairing), `SettingsSectionList` (rowGroup pairing renders paired rows only for adjacent same-group fields), regression checks that global settings no longer expose the fields.
- Full lifecycle e2e suite passes locally on Windows (61/61, ~7m): includes a real-UI test that changes both selects in the Startup Args tab, verifies they persist to the installation record without touching global settings or `config.ini` pre-relaunch, then relaunches and confirms the exact values land in Manager's real `config.ini` (last-match read mirroring configparser), asserts Manager's startup log line `[ComfyUI-Manager] network_mode: <value>` for the exact chosen mode, probes Manager's real v2 API security gate, and confirms the installed torch family is unchanged.

Part of #1037

### CODE REVIEW

File: src/main/lib/managerConfig.ts [hunk 1]
```diff
@@ -28,20 +36,113 @@ function legacyConfigPath(installPath: string): string {
...
+  for (let i = start + 1; i < end; i++) {
+    const current = lines[i] ?? ''
+    if (isSecurityKey(current)) {
+      lines[i] = line + (current.endsWith('\r') ? '\r' : '')
+      return lines.join('\n')
+    }
+  }
+  lines.splice(start + 1, 0, line + ((lines[start] ?? '').endsWith('\r') ? '\r' : ''))
+  return lines.join('\n')
+}
```
[the remark this answers is not in the document]
Comment (Kosinkadink) [reply src/main/lib/managerConfig.ts]: Fixed in 3a1e6b52, with one deliberate deviation: key matching is now case-insensitive and case-variant duplicates are collapsed (configparser's optionxform lowercases option keys, and Manager's migration path parses with strict=True where such duplicates raise). Section matching stays case-sensitive on purpose - configparser section names are case-sensitive and Manager indexes config['default'], so a hand-written [Default] is a section Manager never reads; we append a real [default] instead of editing it. Covered by new withSecurityLevel tests.

File: src/main/settings.ts [hunk 2]
```diff
@@ -218,6 +224,18 @@ const SETTINGS_SCHEMA = {
...
+  // can't leak a free-form string.
+  managerSecurityLevel: {
+    nullable: false,
+    telemetry: {
+      policy: 'value',
+      prop: 'setting_manager_security_level_selected',
+      toTelemetry: (raw) =>
+        raw === 'strong' || raw === 'normal' || raw === 'normal-' || raw === 'weak' ? raw : null,
+    },
+  },
```
[the remark this answers is not in the document]
Comment (Kosinkadink) [reply src/main/settings.ts]: Resolved by removal in 98aefb5a: the Manager security level is now a per-install launch setting (Startup Args tab) rather than a global setting, so the managerSecurityLevel entry in KnownSettings - including this toTelemetry check - no longer exists.

File: e2e/lifecycle.test.ts [hunk 3]
```diff
@@ -975,6 +975,139 @@ test('captures a snapshot for the picker-driven restore test @lifecycle', async
...
+  // touched by the picker edit. Snapshot (rather than assert emptiness)
+  // so a profile reused from a run of the old global design - which may
+  // carry a stale settings.json key - can't flake this.
+  const globalBefore = await ctx.panel.evaluate<string | null>(
+    `window.api.getSetting('managerSecurityLevel')`,
+  )
+
+  // Real entry: running host title pill -> picker Startup Args tab (the
+  // per-install surface this setting lives on since it moved off Global
+  // Settings -> Advanced).
```
[the remark this answers is not in the document]
Comment (Kosinkadink) [reply e2e/lifecycle.test.ts]: Addressed in 85da0d04: rewrote the comments in e2e/lifecycle.test.ts and src/main/lib/ipc/registerSettingsHandlers.test.ts to describe the current per-install behavior (picker edits must not touch the global settings store; the control lives on the Startup Args tab) without referencing the former global placement.

File: src/main/sources/common/launchSettingsFields.test.ts [hunk 4]
```diff
@@ -17,6 +17,25 @@ vi.mock('../../lib/paths', () => ({
...
+  for (const part of key.split('.')) {
+    if (val == null || typeof val !== 'object') return key
+    val = (val as Record<string, unknown>)[part]
+  }
+  return typeof val === 'string' ? val : key
+}
+
+vi.mock('../../lib/i18n', () => ({
+  t: (key: string) => lookup(key),
+}))
```
[the remark this answers is not in the document]
Comment (Kosinkadink) [reply src/main/sources/common/launchSettingsFields.test.ts]: Addressed in 85da0d04: extracted the shared en.json dotted-key resolver into src/main/lib/localeTestHelper.ts (lookupEnMessage); both launchSettingsFields.test.ts and registerSettingsHandlers.test.ts now import it instead of maintaining duplicate resolvers. The mock factories use a dynamic import since vi.mock factories are hoisted. All 2885 unit tests pass.

**Review by MaanilVerma — APPROVED** [review]
